### ISSUE TITLE
Keyboard applet inconsistent size

### ISSUE THREAD

**Issue by HanabishiRecca** [linuxmint/cinnamon#9505]
```
 * Cinnamon version: 4.6.6
 ```

## Issue
Due to font letters width inconsistency, layout switching leads to keyboard applet resize.
E.g. EN and RU have different width in non-monospace font.

**Demonstration**
![layout](https://user-images.githubusercontent.com/13597663/89292709-390f2400-d676-11ea-9d40-f04cf0e00b20.gif)

## Expected behaviour
I think applet width should be consistent.
For example I've made a fast fix: https://github.com/HanabishiRecca/cinnamon/commit/104799129d210738a1cb5e71d0bf0ad253e1b26a

**How it looks**
![layout-fixed](https://user-images.githubusercontent.com/13597663/89293778-e8002f80-d677-11ea-9ec9-120c534abc6c.gif)

Fixed width is a dirty solution and `text-align: center;` doesn't work for whatever reason, but you get the point.

**Comment by smurphos** [comment]
I'm kind of thinking the theme should handle this rather than hard-coding style info in the applet.
Having said that none of the themes on my system do handle it currently. Adding this to my Cinnamox theme solves the resizing issue and look OK and scales OK with different font-sizes.

`.panel-status-button .applet-label { padding-left: 0.2em; width: 1.2em;  }`

**Comment by smurphos** [comment]
Another option might be to enforce monospace.

\this._applet_label.set_style('width: 1.2em;');\

**Comment by HanabishiRecca** [comment]
> If it was going to be patched in the applet something like this would probably be better to properly handle different font sizes

Yeah. Also `text-align: center` might be good but doesn't work for some reason.

> Another option might be to enforce monospace?

There is some drawbacks too, like inconsistent font look. Also for me it breaks vertical alignment:
![image](https://user-images.githubusercontent.com/13597663/89399073-62d65280-d72b-11ea-9a00-3bfafba58cb5.png)

**Comment by MinePro120** [comment]
I can reproduce, in my case it's the calendar applet, that I've configured to show seconds as well. Changing the font (via editing my theme's cinnamon.css) breaks vertical alignment as stated above.

**Comment by MinePro120** [comment]
> Yeah. Also `text-align: center` might be good but doesn't work for some reason.
I don't know CSS at all, apart from what I learned trying to resolve this, but I think the problem is that the label inherits the text's actual width, which in the case of non-mono fonts, changes. The text itsleft is aligned inside the label. No idea why some fonts break vertical allignment.
